app/controllers/organisation_relationships_controller.rb

```diff
@@ -66,6 +66,20 @@ def create!(child_organisation_id:, parent_organisation_id:, relationship_type:)
     @resource.save!
   end
 
+  def remove_housing_provider
+    render "organisation_relationships/remove_housing_provider", layout: "application"
+  end
+
+  def delete_housing_provider
+    organisation_relationship_to_remove_id = OrganisationRelationship.find_by(child_organisation_id: @organisation.id, parent_organisation_id: organisation_to_remove_id, relationship_type: OrganisationRelationship::OWNING).id
+    delete(organisation_relationship_to_remove_id)
+    redirect_to housing_providers_organisation_path(removed_organisation_id: organisation_to_remove_id)
+  end
+
+  def delete(organisation_relationship_to_remove_id)
+    OrganisationRelationship.destroy(organisation_relationship_to_remove_id)
```

Comment:
prefer `destroy!` with a bang so that it'll raise an error when it's not successful